.github/workflows/nightly.yaml

```diff
@@ -34,14 +34,31 @@ permissions: read-all
 jobs:
   cirq-stable:
     name: Nightly Pytest Cirq stable
-    uses: ./.github/workflows/nightly-pytest.yaml
+    uses: ./.github/workflows/_nightly-cirq-test.yaml
     with:
       args: ''
       reason: '(nightly)'
+      cirq-release: 'stable'
 
   cirq-pre:
     name: Nightly Pytest Cirq pre-release
-    uses: ./.github/workflows/nightly-pytest.yaml
+    uses: ./.github/workflows/_nightly-cirq-test.yaml
     with:
       args: '--pre'
       reason: '(nightly)'
+      cirq-release: 'pre-release'
+
+  osv:
+    name: Nightly OSV code scan
+    uses: ./.github/workflows/osv-scanner.yaml
+    permissions: write-all
```

Comment:
Can this and the scorecard use the same permissions as in the referenced workflows?

Alternatively, you could just add the daily `schedule:` trigger to osv-scanner.yaml and scorecard-scanner.yaml so you don't need to maintain permissions in two places.

Reply:
> Can this and the scorecard use the same permissions as in the referenced workflows?
> 
> Alternatively, you could just add the daily `schedule:` trigger to osv-scanner.yaml and scorecard-scanner.yaml so you don't need to maintain permissions in two places.

Thanks for the feedback. You're right that avoiding a controlling workflow (nightly.yaml) avoids some duplication, simplifies permissions, and is easier to maintain. I was seeking to put some organization on the whole scheme so that people had an easier time understanding the relationships and how different pieces are triggered. But it is turning into more work than expected.

I'll remove nightly.yaml and update the other workflows to do the schedules themselves.